browser/src/control/Control.JSDialog.js

```diff
@@ -606,6 +606,16 @@ L.Control.JSDialog = L.Control.extend({
 				// Call this.close(key, true) for the current dialog
 				this.close(autoFilterDialogId, true);
 			}
+
+			if (autoFilterDialogId === 'autoFillPopup') {
```

Comment:
this is no longer needed as we fixed problem with `isAutofilter`

Reply:
or am I wrong?

Reply:
This is needed. I don't want to put `closePopup` manually everywhere when there is a need to close popup on tab change. Here is easy, just don't close the autoFilter popup only if we drag with autoFilterMarker. otherwise close it. :)

Reply:
ah, it's needed... then please add `instance.isAutoFillPopup` property and detect it here. (We want to avoid using the hardcoded id in many places).
Also change name of the function to `closePopupsOnTabChange` so it will tell the truth.

Reply:
Thanks ! 
Done